impl/src/main/java/org/ehcache/internal/store/heap/OnHeapStore.java

```diff
@@ -174,7 +176,8 @@ public OnHeapStore(final Configuration<K, V> config, TimeSource timeSource, Copi
   }
   
   private OnHeapValueHolder<V> internalGet(final K key, final boolean updateAccess) throws CacheAccessException {
-    return map.computeIfPresent(key, new BiFunction<K, OnHeapValueHolder<V>, OnHeapValueHolder<V>>() {
+    accessObserver.begin();
```

Comment:
When the `OnHeapStore` acts as anything but an authority, this method is never called, so no need to involve the tier specific stat.